src/crimson/os/seastore/lba_manager.h

```diff
@@ -146,6 +145,37 @@ class LBAManager {
     laddr_t addr,
     int delta) = 0;
```

Comment:
This seems to be used by the UT test_transaction_manager

Reply:
Not able to see the user, can you point to the code?

Reply:
https://github.com/ceph/ceph/blob/fab66be49ffed7b42a2c4b93db7f38c23334d925/src/test/crimson/seastore/test_transaction_manager.cc#L689-L691

Reply:
Doesn't look like it is calling `ref_ret incref_extent(t, addr, delta)`, right ?

Reply:
But `TM::inc_ref` will call `LBAManager::incref_extent()`.

Reply:
But it doesn't call the version of `ref_ret LBAManager::incref_extent(t, addr, delta)`

Reply:
Ah, yes, will remove it:-)

Reply:
Um, `ref_ret LBAManager::incref_extent(t, addr, delta)` is called by the `LBAManager::remap_mappings()` in this PR. Although we only remap at most two mappings for now, which means we can call `ref_ret LBAManager::incref_extent(t, addr)` instead of `ref_ret LBAManager::incref_extent(t, addr, delta)`, but I think maybe we can keep the way it is in case that we remap more than two mappings in the future. What do you think?

Reply:
`BtreeLBAManager::remap_mappings()` is reusing `BtreeLBAManager::incref_extent(t, addr, delta)`, which is fine, and should already support to remap more than 2 mappings IIUC.

But I don't see why `incref_extent(t, addr, delta)` is necessary to be exposed at the level of `LBAManager`.